scout/load/panel.py

```diff
@@ -151,6 +173,47 @@ def load_panelapp_panel(adapter, panel_id=None, institute="cust000", confidence=
             raise err
 
 
+def load_panelapp_green_panel(adapter, institute):
+    """Load/Update the panel containing all Panelapp Green genes
+
+    Args:
+        adapter(scout.adapter.MongoAdapter)
+        institute(str): _id of an institute
+
+    """
+    LOG.info("Fetching all panel app panels")
+    panel_ids = _panelapp_panel_ids()
+
+    # check and set panel version
+    PANEL_NAME = "PANELAPP-GREEN"
+    old_panel = adapter.gene_panel(panel_id=PANEL_NAME)
+    green_panel = {
+        "panel_name": PANEL_NAME,
+        "display_name": "PanelApp Green Genes",
+        "institute": institute,
+        "version": old_panel["version"] + 0.1 if old_panel else 1,
+        "date": datetime.now(),
+    }
+
+    genes = set()  # avoid duplicate genes from different panels
+    # Loop over all PanelApp panels
+    for _ in panel_ids:
+        # And collect their green genes
+        parsed_panel = _parse_panelapp_panel(adapter, _, institute, "green")
+        genes.update({(gene["hgnc_id"], gene["hgnc_symbol"]) for gene in parsed_panel.get("genes")})
```

Comment:
This is a workaround because we want unique genes but you dicts are unhashable and can't be added to a set